src/main/java/org/opentripplanner/graph_builder/module/osm/OsmDatabase.java

```diff
@@ -1086,6 +1087,24 @@ private void processRoad(OSMRelation relation) {
     }
   }
 
+  private int getLevel(OSMWithTags o) {
```

Comment:
This should be part of  `OSMWithTags`.

Also, please remove the duplication in the method.